Block/RecentPostsBlockService.php

```diff
@@ -34,6 +34,11 @@ class RecentPostsBlockService extends BaseBlockService
     protected $manager;
 
     /**
+     * @var Pool
+     */
+    private $adminPool;
```

Comment:
I ment this parameter, was previously before, so it was accesible without calling any getter. It's the same as this:

https://github.com/sonata-project/SonataAdminBundle/blob/3.x/Controller/CRUDController.php#L1381

In this case we were relying on a parameter that wasn't declared, and someday one guy added that as a protected and our app broke. I don't think this one is this critical but maybe we could add some magic __get with a deprecated message?